libpod/network/netavark/config.go

```diff
@@ -102,7 +103,36 @@ func (n *netavarkNetwork) networkCreate(newNetwork types.Network, defaultNet boo
 				}
 
 			default:
-				return nil, errors.Errorf("unsupported network option %s", key)
+				return nil, errors.Errorf("unsupported bridge network option %s", key)
+			}
+		}
+	case types.MacVLANNetworkDriver:
+		if newNetwork.Internal {
+			return nil, errors.New("internal is not supported with macvlan")
+		}
+		if newNetwork.NetworkInterface != "" {
+			interfaceNames, err := internalutil.GetLiveNetworkNames()
+			if err != nil {
+				return nil, err
+			}
+			if !util.StringInSlice(newNetwork.NetworkInterface, interfaceNames) {
+				return nil, errors.Errorf("parent interface %s does not exists", newNetwork.NetworkInterface)
+			}
+		}
+		if len(newNetwork.Subnets) == 0 {
+			return nil, errors.Errorf("macvlan driver needs at least one subnet specified, DHCP is not supported with netavark")
```

Comment:
I just wonder if an end-user will be confused by "netavark" in this message. Maybe "Podman's network stack" instead?

Reply:
No netvark will at least bring up search results. "Podman's network stack" can mean anything even CNI.

Reply:
Is this forever or do we plan on adding dhcp support at some point?

Reply:
we plan to support this but it is not trivial, basically needs another deamon